package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "testpilot-metrics",
+  "version": "0.0.1",
+  "description": "Metrics library for Test Pilot experiments. Sends events to Google Analytics always, and Test Pilot metrics pipeline while the experiment is an active Test Pilot experiment.",
+  "main": "index.js",
+  "scripts": {
+    "test": "./node_modules/.bin/mocha test/*",
+    "build-docs": "./node_modules/.bin/documentation build testpilot-metrics.js -f md -o API.md"
```

Comment:
Techically, `./node_modules/.bin/` should be added to the `$PATH` when using `npm run ...`, so you _should_ be able to simplify these to `mocha test/*` and `documentation build ...` (without the node_modules prefixes).